config/sidekiq_scheduler.yml

```diff
@@ -2,3 +2,7 @@ CreateDailySpoolFiles:
   cron: "0 3 * * * America/New_York"
   class: EducationForm::CreateDailySpoolFiles
   description: "Generate a daily set of spool files and transmit them over SFTP to a backend system"
+
+SidekiqStatsJob:
+  every: 1m
```

Comment:
Can you test locally and ensure that this won't be run multiple times per minute - one per Scheduler instance? https://github.com/moove-it/sidekiq-scheduler/commit/ee317fd427bf600ba2562f9ae3cc64b9880d0e68 makes me think that only `cron` and `at` type jobs are de-duplicated.